cli/src/merge_tools/mod.rs

```diff
@@ -107,7 +107,7 @@ pub fn run_mergetool(
 
     let editor = get_merge_tool_from_settings(ui, settings)?;
     match editor {
-        MergeTool::Internal => unimplemented!("run_mergetool with internal mergetool"),
+        MergeTool::Builtin => unimplemented!("run_mergetool with internal mergetool"),
```

Comment:
Also change the message here and below? Also, "run_mergetool" sounds too implementation-detaily for a message to the user

Reply:
Updated `internal` -> `builtin`, but this code path isn't accessible until https://github.com/martinvonz/jj/pull/2120, at which point they've been implemented, so users shouldn't see it regardless.